_posts/en/2020-11-17-FWD50-in-the-open.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  "FWD50 in the open"
+ref: FWD50-2020
+lang: en
+author: "Rachel Muston and Guillaume Charest"
+date: "2020-11-20"
```

Comment:
that date is in the future!! Jekyll gave me hard time when I wanted to read the post in a html-rendered version

Reply:
Haha, good catch, will update :smile: